### PR TITLE
chore: Added ru locale, added voximplant adapter, groq provider, switched to whisper in deepgram (multilang)

### PR DESCRIPTION
**Pull request by iowathe3rd**
bunch of chore code, not serius

### CODE REVIEW

File: bolna/providers.py [hunk 1]
```diff
@@ -42,7 +43,7 @@
     'anyscale': LiteLLM,
     'custom': OpenAiLLM,
     'ola': OpenAiLLM,
-    'groq': LiteLLM,
+    'groq': GroqLLM,
```
Comment (prateeksachan) [inline bolna/providers.py:46]: any specific reason of not using litellm for groq?

File: local_setup/requirements_bolna.txt [hunk 2]
```diff
@@ -0,0 +1,28 @@
+aiobotocore==2.9.0
```
Comment (prateeksachan) [inline local_setup/requirements_bolna.txt:1]: these packages could be installed when installing bolna package in the docker

File: local_setup/telephony_server/voximplant_api_server.py [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+        if not call_details or "recipient_phone_number" not in call_details:
+            raise HTTPException(status_code=404, detail="Recipient phone number not provided")
+
+        app_callback_url, websocket_url = populate_ngrok_tunnels()
+
+        print(f'app_callback_url: {app_callback_url}')
+        print(f'websocket_url: {websocket_url}')
+        print(f'recipient_phone_number: {recipient_phone_number}')
+
+        voxapi = VoximplantAPI("credentials/voximplant.json")
```
Comment (prateeksachan) [inline local_setup/telephony_server/voximplant_api_server.py:56]: can you also please commit a test/temporary sample voximplant json file.

File: bolna/providers.py (hunk 1 above)
Comment (iowathe3rd) [reply bolna/providers.py:46]: Hi, I'm sorry for the very dirty pr, I'll correct it and write in the comments

File: local_setup/telephony_server/voximplant_api_server.py (hunk 3 above)
Comment (iowathe3rd) [reply local_setup/telephony_server/voximplant_api_server.py:56]: Integration with Voximplant should be two-way, unfortunately there is no built-in ability to stream audio from a websocket, but on the bolna side everything is ok

File: local_setup/requirements_bolna.txt (hunk 2 above)
Comment (iowathe3rd) [reply local_setup/requirements_bolna.txt:1]: I almost missed the mark, I'm not that good at python

File: bolna/providers.py (hunk 1 above)
Comment (iowathe3rd) [reply bolna/providers.py:46]: > any specific reason of not using litellm for groq?

in cases when user need to add more then 1 litellm driven provider, 

real case example:
I could be wrong, but in our project, which is a fork of your repository, we need to use both groq and antrophic at the same time, from env variables

**Pull request closed without merging**